doc/en/how-to/usage.rst

```diff
@@ -26,6 +26,11 @@ Pytest supports several ways to run and select tests from the command-line or fr
 
     pytest test_mod.py
 
+When passing a path as a positional argument, pytest does not recursively
```

Comment:
IMHO this is a bit verbose, I think it is enough to mention that the user can pass the absolute or relative path to the test file.

Something like this:


```rst
**Run tests in a module**

You can pass the relative or the absolute path to a test module directly:

.. code-block:: bash

    pytest test_mod.py
```

I don't feel the need to mention that pytest will not look for `test_mod.py` recursively, few tools work like that (without a separate flag that is).

(I would leave a *suggestion* but GH does not let me place a suggestion outside the diff)

Reply:
@The-Compiler, which teaches pytest for a living, is this something users in general get confused about?

Reply:
I don't think this ever came up, no. FWIW, here is the slide I have for it:

![image](https://github.com/pytest-dev/pytest/assets/625793/8570ea32-e385-4313-95bd-ec25df101ccd)

I agree that it's not really the job of the pytest documentation to explain this in detail. There's nothing special about pytest here, any other tool that takes a file name would work the exact same way.

@brycegbrazen can you elaborate a bit on why you think this is necessary, or why you would consider that pytest would do anything like this in the first place?